cypress.json

```diff
@@ -16,6 +16,7 @@
     "password": "admin",
     "ENDPOINT_WITH_PROXY": false,
     "MANAGED_SERVICE_ENDPOINT": false,
-    "VISBUILDER_ENABLED": false
+    "VISBUILDER_ENABLED": false,
+    "DATASOURCE_MANAGEMENT_ENABLED": false
```

Comment:
The default value is false, and I didn's see this being used anywhere. Do we really need it?

Reply:
Look at files:

 [workflows/cypress-workflow-bundle-snapshot-based.yml](https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/325/files#diff-6bca3f4633639fc3b43e7344e31d116d04654ee9c21f376395abfb1772b7ee49)

[dashboards/opensearch-dashboards/datasource-management-plugin/2_datasource_table.spec.js](https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/325/files#diff-c3c0c692938276a0e5ccbcfe5491390149ac8c111ed67f6a5d1d94e7b1fd3a85)